RELEASE_NOTES.md

```diff
@@ -1,6 +1,13 @@
 Authentication Service MKII release notes
 =========================================
 
+0.2.4
+-----
+* CONFIGURATION CHANGE - the templates directory is now configurable. Add
+  `template-dir = templates` to any existing configuration files to preserve current
+  behavior. This purpose of this change is primarily to allow other applications to
```

Comment:
The purpose of this change...

Reply:
Fixed locally. It'll go out in my next PR, and meanwhile can stay to mark my shame.